jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
+  ]
```

Comment:
This may be okay on Windows, I don't know.  `consul_agent` used to work this way for a while, but we ran into major problems with this setup, never fully got to the bottom of the issue, but switching consul_agent to do recursing fixed it.

These issues happened a year ago with customers and open source users (https://lists.cloudfoundry.org/archives/list/cf-dev@lists.cloudfoundry.org/thread/LLUP4V5DGOU2KMLZWDTUWPTHWOOL4UHH/#LLUP4V5DGOU2KMLZWDTUWPTHWOOL4UHH).

Now, with dnsmasq, consul does not do recursing, but dnsmasq does I believe.

Configurable DNS is on master, but it's not in an official release yet, and their release cadence is fairly slow.  There is no large payload fix in Consul's codebase anywhere.